github/github.go

```diff
@@ -681,6 +683,36 @@ func (r *ErrorResponse) Error() string {
 		r.Response.StatusCode, r.Message, r.Errors)
 }
 
+func (r *ErrorResponse) Is(target error) bool {
```

Comment:
Please add a comment for these new methods... something like:

```
// Is tests if the target error is of type ErrorResponse.
```

Reply:
Done.